app/controllers/devise_token_auth/omniauth_callbacks_controller.rb

```diff
@@ -235,7 +235,11 @@ def get_resource_from_auth_hash
       @uid = auth_hash['uid']
       @provider = auth_hash['provider']
       @email ||= auth_hash['info']['email']
-      rc = resource_class
+      if scoped_omniauth_provider? @provider
```

Comment:
We only want to use the active scope on users from the linked provider. Salesforce will not have the scope applied. This is configured in the BTG api with:
`DeviseTokenAuth.scoped_omniauth_providers`